src/Windows/Avalonia.Win32/DxgiSwapchain/HANDLE.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Win32.DxgiSwapchain
+{
+#nullable enable
+    public unsafe struct HANDLE
```

Comment:
IIRC there were some potential problems on some platforms (either arm or 32-bit x86) with marshalling structures as return values. Which is why we are usually just using `IntPtr` or `int` for P/Invoke.

Reply:
> IIRC there were some potential problems on some platforms (either arm or 32-bit x86) with marshalling structures as return values. Which is why we are usually just using `IntPtr` or `int` for P/Invoke.

Yes, that struct has a `void*` as its internal value which should have the same size implications as `IntPtr` insofar as it varies from system to system.